packages/client/src/__tests__/integration/happy/exhaustive-schema/__snapshots__/test.ts.snap

```diff
@@ -7228,6 +7307,10 @@ export namespace Prisma {
    * ManyRequired createMany
    */
   export type ManyRequiredCreateManyArgs = {
+    /**
+     * The data used to create many ManyRequireds.
```

Comment:
The pluralize is a bit weird and we removed it from most places but not here apparently, and `many` in front would be sufficient I think to indicate it's more than one.

Reply:
I guess we would need to remove pluralize from `getArgFieldJSDoc`